### ISSUE TITLE
Modkit allows to have duplicate sections of same module config

### ISSUE THREAD

**Issue by il10241024** [constructorfabric/gears-rust#3003]
> 🔗 **Mirrored from** [cyberfabric/cyberware-rust#1255](https://github.com/cyberfabric/cyberware-rust/issues/1255)
> Originally opened by **MikeFalcon77** on 2026-03-20T00:31:28Z

---

If you add a module section with a duplicated name to the config, it will still work somehow:
```yaml
modules: 
     module1: 
     module2:
     module1:
```

It should generate an error.     

---
